cmd/snap/cmd_run.go

```diff
@@ -33,10 +34,12 @@ import (
 
 type cmdRun struct {
 	Positional struct {
-		SnapApp string `positional-arg-name:"<app name>" description:"the application to run, e.g. hello-world.env"`
+		SnapApp string `positional-arg-name:"<app name>" description:"the snap (e.g. hello-world) or application to run (e.g. hello-world.env)"`
 	} `positional-args:"yes" required:"yes"`
 
-	Command string `long:"command" description:"alternative command to run"`
+	Command  string `long:"command" description:"alternative command to run"`
+	Hook     string `long:"hook" description:"run a specific hook contained within the snap"`
+	Revision string `long:"revision" description:"use a specific snap revision instead of the active one (this only applies when using --hook)"`
```

Comment:
Now that we know we can hide options, would this be better if we hid it? We can of course support `15/hook-name` instead, but it leads to more complex code and at least to me seems to obfuscate what's happening. It's up to you.